src/BenchmarksServer/Startup.cs

```diff
@@ -217,6 +221,28 @@ public static int Main(string[] args)
                                 Debug.Assert(process == null);
                                 process = StartProcess(hostname, database, sqlConnectionString, Path.Combine(tempDir, benchmarksDir),
                                     job, dotnetHome);
+                                
+                                if (timer != null)
```

Comment:
I don't think this is necessary since the timer should be disposed an set to null when deleting the job.